src/assets/build_assets.rs

```diff
@@ -328,10 +345,31 @@ fn dependencies_from_pattern(pattern: &Pattern) -> Vec<OtherSyntax> {
     .collect()
 }
 
+/// Removes any subscope from the scope.
```

Comment:
It may be clearer to use syntect terminology here. Maybe, "Removes any context name from the syntax reference"? And then to rename "subscope" elsewhere to "context" (or "subcontext" - without a `#something`, `#main` is implied)

Reply:
That is definitely a good idea! Thanks for pointing that out. I have not internalized syntect terminology enough to fluently speak it yet :)

I pushed a commit that I think improves the terminology. Looking forward to your thoughts on the new code.

Reply:
Looks great - thanks!